xmodule/tests/test_video.py

```diff
@@ -916,6 +916,12 @@ def test_student_view_data_with_hls_flag(self, mock_get_video_info, mock_get_vid
             'v': 'set_youtube_id_of_11_symbols_here',
         },
     },
+
```

Comment:
`test_video.py` has legacy tests dealing with youtube transcripts that are complementary to this PR. Because these legacy tests override settings, they need to explicitly offer the updates to the YOUTUBE environment variable added as part of this work to continue working.